app/views/album_recipient_mailer/share_private_album_to_recipient_mail.html.erb

```diff
@@ -0,0 +1,7 @@
+<p>Dear Client,</p>
+<p>Please check your photos by clicking on the View Album link. It will ask you to enter the passcode. Please apply provided passcode in this email.</p>
+<p> <%= link_to "View Album", album_url(@album) %> </p>
+<p> <%= @album_recipient.last.custom_message %> </p>
+<p> passcode: <%= @album_passcode %> </p>
```

Comment:
Here is url for view album `localhost:3000/albums/:album_id?token=contact_token`, please update url.